core/src/main/java/org/mobilitydata/gtfsvalidator/input/CountryCode.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.input;
+
+import com.google.common.collect.ImmutableSet;
+import java.util.Locale;
+import java.util.Set;
+
+/** Represents the country code of a GTFS feed. */
+public class CountryCode {
+  private static final Set<String> ISO_COUNTRIES = ImmutableSet.copyOf(Locale.getISOCountries());
+  private static final String INVALID_COUNTRY_CODE_MESSAGE =
+      "Country code must follow format `cc`, where cc is an ISO Alpha 2 country code";
+  // ISO 3166-1 Alpha 2 country code for the United Kingdom is "GB" but feed names may use "UK".
+  private static final String UK = "UK";
+  private static final String GB = "GB";
+  private final String countryCode;
```

Comment:
I would use Java `Optional` here for `countryCode` instead of storing `null` directly to avoid potential future NPEs:
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html